apps/manuscripts/search_indexes.py

```diff
@@ -28,9 +33,37 @@ def prepare_image_availability(self, obj):
     def prepare_catalogue_numbers(self, obj):
         return obj.historical_item.get_catalogue_numbers_display()
 
+        # Extract and serialize the `Date` model fields
+
+    def prepare_date(self, obj):
+        if obj.historical_item and obj.historical_item.date:
+            return obj.historical_item.date.date  # Returns the `date` string (e.g., "1000x1001")
+        return None
+
+    def prepare_date_min(self, obj):
+        if obj.historical_item and obj.historical_item.date and obj.historical_item.date.min_weight:
+            return obj.historical_item.date.min_weight
+        return 0  # Default value
+
+    def prepare_date_max(self, obj):
+        if obj.historical_item and obj.historical_item.date and obj.historical_item.date.max_weight:
```

Comment:
```suggestion
        if obj.historical_item.date and obj.historical_item.date.max_weight:
```
Because we're sure historical item will never be Null